src/Components/Search/HoverItemView/HoverItemView.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import './HoverItemView.scss';
+
+function HoverItemView({ item, showProduct }) {
+  const { name, imageUrl } = item;
+
+  const navigate = useNavigate();
+  function moveItemPage() {
+    navigate(`/productDetail/${item.id}`);
+  }
+
+  return (
+    <div
+      className="hoverItemView"
+      onMouseEnter={() => {
+        showProduct();
+      }}
```

Comment:
```suggestion
      onMouseEnter={showProduct}
```
콜백을 사용할 이유가 없어 보입니다!